<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AM Modulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bg_window">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>900</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>900</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>900</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>src/img/mainwindow_bg.PNG</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="carrier_type_1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>415</y>
      <width>114</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Choose the type of wave of the carrier</string>
    </property>
    <item>
     <property name="text">
      <string>Cosine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sine</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spin_freq_1">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>415</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Choose the frequency of the Carrier</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
    </property>
    <property name="suffix">
     <string>Hz</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_freq_2">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>865</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Choose the frequency of the Carrier</string>
    </property>
    <property name="suffix">
     <string>Hz</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
   <widget class="QComboBox" name="carrier_type_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>865</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Choose the type of wave of the carrier</string>
    </property>
    <item>
     <property name="text">
      <string>Cosine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sine</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_record_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>415</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Record 5 seconds audio to transmit</string>
    </property>
    <property name="text">
     <string>Record Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load_1">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>415</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Load audio file from disk</string>
    </property>
    <property name="text">
     <string>Load Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_record_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>865</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Record 5 seconds audio to transmit</string>
    </property>
    <property name="text">
     <string>Record Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load_2">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>865</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Load audio file from disk</string>
    </property>
    <property name="text">
     <string>Load Audio</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_save_charts">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>445</y>
      <width>110</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Charts</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_save_audio">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>465</y>
      <width>110</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Audios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>415</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Save files</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_play">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1015</x>
      <y>415</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reproduce resulting modulated signal</string>
    </property>
    <property name="text">
     <string>Play Signal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_save_charts_2">
    <property name="geometry">
     <rect>
      <x>1015</x>
      <y>445</y>
      <width>110</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Loop</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_real_time_plot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_audio1_freq">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>225</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_audio2_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>485</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_audio2_freq">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>675</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_carrier1_freq">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>225</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_carrier2_time">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>485</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_carrier2_freq">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>675</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_carrier1_time">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>35</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_modsig2_freq">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>675</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_modsig2_time">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>485</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_modsig1_freq">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>225</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_modsig1_time">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>35</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_modsigf_time">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>35</y>
      <width>360</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_modsigf_freq">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>225</y>
      <width>360</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QListWidget" name="console_display">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>495</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
font: 8pt &quot;Consolas&quot;;
color: rgb(0, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="autoScrollMargin">
     <number>50</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::Batched</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <property name="uniformItemSizes">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="link_to_docs">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>855</y>
      <width>285</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click to see documentation on GitHub</string>
    </property>
    <property name="text">
     <string>Documentation</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/img/doc.png</normaloff>src/img/doc.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
